### PR TITLE
Add notes about booting linux kernel

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: binary_transparency/firmware/devices/usb_armory/README.md [hunk 1]
```diff
@@ -36,3 +36,89 @@ directory:
...
+
+There are some invariants which must hold for this chain to work:
+1. The `firmware` partition MUST be located at the precise offset mentioned
+    above.
+2. The `firmware` partition MUST be formatted with ext4.
+3. The `firmware` partition MUST contain a `/boot` directory with at least the
+    following contents:
+    * `armory-boot.conf` - a JSON file which tells the bootloader which files
+      to load.
+    * a valid ARM linux Kernel image
```
Comment (mhutchinson) [inline binary_transparency/firmware/devices/usb_armory/README.md:54]: nit this bullet doesn't end with a period and the others do. Recommend removing period on the others, but you can add one to this if that's your jam.

File: binary_transparency/firmware/devices/usb_armory/README.md [hunk 2]
```diff
@@ -36,3 +36,89 @@ directory:
...
+    * a valid DTB file.
+   Note that the `armory-boot.conf` file also contains SHA256 hashes of the
+   kernel and DTB files, and these MUST be correct.
+
+
+> :frog: The [Armory Debian Base Image](https://github.com/f-secure-foundry/usbarmory-debian-base_image/releases)
```
Comment (mhutchinson) [inline binary_transparency/firmware/devices/usb_armory/README.md:60]: Not sure I get the frog reference?

File: binary_transparency/firmware/devices/usb_armory/README.md [hunk 3]
```diff
@@ -36,3 +36,89 @@ directory:
...
+> is a good source for the kernle (zImage) and dtb files.
```
Comment (mhutchinson) [inline binary_transparency/firmware/devices/usb_armory/README.md:61]: sp: kernel

File: binary_transparency/firmware/devices/usb_armory/README.md [hunk 4]
```diff
@@ -36,3 +36,89 @@ directory:
...
+>
+> You can decompress and monut the image to access the files like so:
```
Comment (mhutchinson) [inline binary_transparency/firmware/devices/usb_armory/README.md:63]: Is `monut` a mounted donut?

**Review by mhutchinson — APPROVED** [review]

File: binary_transparency/firmware/devices/usb_armory/README.md (hunk 1 above)
Comment (AlCutter) [reply binary_transparency/firmware/devices/usb_armory/README.md:54]: ta, done

File: binary_transparency/firmware/devices/usb_armory/README.md (hunk 2 above)
Comment (AlCutter) [reply binary_transparency/firmware/devices/usb_armory/README.md:60]: no reference, just wanted "friendly tip" emoji, but couldn't see one so I chose a friendly frog.

File: binary_transparency/firmware/devices/usb_armory/README.md (hunk 3 above)
Comment (AlCutter) [reply binary_transparency/firmware/devices/usb_armory/README.md:61]: d'oh, done

File: binary_transparency/firmware/devices/usb_armory/README.md (hunk 4 above)
Comment (AlCutter) [reply binary_transparency/firmware/devices/usb_armory/README.md:63]: it should be if it isn't already...

**Pull request merged**